### PR TITLE
Restyled the file button

### PR DESCRIPTION
**Pull request by bliker**
I went ahead and removed the old code and just added simple border on the side and hover effect. 
screenshot 2016-06-19 12 28 02

### CODE REVIEW

**Comment by bliker** [comment]
I think thats more clear and could close #67 ?

**Comment by jasongrout** [comment]
Personally, I *really* like the idea from #67 of not having + and the weird up arrow icon, but rather "New" and "Upload". I think that is much clearer.

**Comment by bliker** [comment]
I am all for it, I will take a look at it tomorrow!

**Comment by bliker** [comment]
I think I reached something that looks right
screenshot 2016-07-05 11 17 48

**Comment by SylvainCorlay** [comment]
@bliker you probably need all font sizes to be 14px for font-awesome pixel perfection.

**Comment by bliker** [comment]
yup, fixed and I also made sure it works on top 
screenshot 2016-07-05 12 12 52

**Comment by bliker** [comment]
I am not happy with the number of horizontal lines in that section but it is ready to merge from my perspective!  🏁

**Comment by SylvainCorlay** [comment]
👍 I also preferred in at the bottom.

File: src/filebrowser/theme.css [hunk 1]
```diff
@@ -34,60 +34,29 @@
...
   flex: 1 1 auto;
   max-width: 100px;
-  padding: 4px 6px;
+  margin: 0;
+  padding: 7px;
   color: #757575;
   background: #FAFAFA;
   border: none;
-  font-size: 16px;
+  font-size: 13px;
```
Comment (SylvainCorlay) [inline src/filebrowser/theme.css]: you probably want to remove that property from theme.css

**Comment by ellisonbg** [comment]
I also like having text, rather than icons. But we have decided that the file browser will need more of a full blown toolbar with more click driven UI controls for other actions (Delete, Shutdown, etc.). 

I also very much agree that we have too many horizontal lines in this section. I will talk with our designers about this CC @matt-bowers

**Comment by ellisonbg** [comment]
Hi, right now we have a bunch of different design work going on that will affect this area of the file buttons. Some of the things you have done here are going to inform that work, but I don't think we are going to merge this particular design change. Some of the planned changes:

* Move to material design icons
* Move away from the lots of horizontal lines
* Move towards a more full featured toolbar with other actions related to files that are selected (rename, delete, etc) to make  the UI more discoverable
* Text rather than icons based buttons where possible...

Closing...

**Pull request closed without merging**